### DISCUSSION TITLE
MSCS%40UIUC/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#86] [Announcements]
# MSCS%40UIUC/

https://csmsapp.github.io/MSCS%40UIUC/

**Comment by pat-jj** [comment]
不一定要面试

**Comment by CFC87** [comment]
今年有申UIUC CS PhD被拒，然后和老师联系答应给一个MSCS然后再补申MSCS并录取的case

**Comment by Alisia-182** [comment]
请问poi是什么意思呀

**Reply by Xinyu-Li-123** [reply]
Professor of Interest，应该是指你想跟着做研究的教授。

**Comment by xichenpan** [comment]
professor of interest

Jinyi Wan ***@***.***> 于 2023年7月25日周二 05:59写道：

**Reply by cchowc** [reply]
TA是什么意思啊

**Comment by TomekWei** [comment]
jeff说的, 他在委员会里:
Here are some statistics about graduate applications in computer scienceat UlUC, for the Fall 2023 admissions cycle:
We received 2427 applications for the MSCS (thesis masters)program.We made 103 admission offers, of which 58 were accepted. So the acceptance rate was 4.2%, and the yield rate was 56.3%.
We received 1537 applications for our PhD program. We mad 255 admission offers, of which 100 were accepted. So theacceptance rate was 17.0%, and the yield rate was 39.2%.
